code/user_settings.py

```diff
@@ -58,3 +58,19 @@ def get_list_from_csv(
             mapping[spoken_form] = output
 
     return mapping
+
+def append_to_csv(filename: str, rows: Dict[str, str]):
+    path = SETTINGS_DIR / filename
+    assert filename.endswith(".csv")
+
+    with open(str(path), "r") as file:
+        line = None
+        for line in file:
+            pass
+        needs_newline = line is not None and not line.endswith("\n")
+    with open(path, "a", encoding="utf-8", newline="") as file:
+        if needs_newline:
+            file.write("\n")
```

Comment:
csv.writer uses the Excel dialect by default which always uses `\r\n` line endings regardless of platform. Consider using `csv.writerow([])`.

Reply:
Done and tested on Windows.